src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -1024,7 +1025,7 @@
     (when (or existing-cert? existing-csr?)
       (let [status (if existing-cert?
                      (if (utils/revoked?
-                           (utils/pem->crl cacrl) (utils/pem->cert cert))
+                           (first (utils/pem->crls cacrl)) (utils/pem->cert cert))
```

Comment:
Will we have to do something similar for `pem->cert`? I seem to remember we thought it could already load multiple, but if that's true, it might be nice to also update the names there to make it clearer.

Reply:
Yeah, I think we added a `pem->ca-cert` that does the `(first (pem->certs cacert))`. I can add a `pem->ca-crl` or `pem->leaf-crl` that encapsulates this.